### PR TITLE
drop not needed numpy packages from release image

### PR DESCRIPTION
**Pull request by dtrawins**
### 🛠 Summary

drop not needed numpy packages from release image

### 🧪 Checklist

- [ ] Unit tests added.
- [ ] The documentation updated.
- [ ] Change follows security best practices.
``

### CODE REVIEW

File: Dockerfile.redhat [hunk 1]
```diff
@@ -488,8 +488,10 @@ RUN if [ "$NVIDIA" == "1" ]; then true ; else exit 0 ; fi ; echo "installing cud
 ENV LD_LIBRARY_PATH=/ovms/lib
 
 COPY --from=pkg /ovms_release /ovms
-COPY --from=build /usr/local/lib/python3.*/site-packages /ovms/python_deps
-COPY --from=build /usr/local/lib64/python3.*/site-packages /ovms/python_deps
+COPY --from=build /usr/local/lib/python3.*/site-packages/jinja2* /ovms/python_deps/
+COPY --from=build /usr/local/lib64/python3.*/site-packages/MarkupSafe-2.1.5.dist-info /ovms/python_deps/
+COPY --from=build /usr/local/lib64/python3.*/site-packages/markupsafe /ovms/python_deps/
```
Comment (mzegla) [inline Dockerfile.redhat:493]: not sure if dist info is required

File: Dockerfile.ubuntu [hunk 2]
```diff
@@ -487,7 +487,9 @@ RUN if [ "$NVIDIA" == "1" ]; then true ; else exit 0 ; fi ; echo "installing cud
     apt-get clean && rm -rf /var/lib/apt/lists/* && rm -rf /tmp/*
 
 COPY --from=pkg /ovms_release /ovms
-COPY --from=build /usr/local/lib/python3.*/dist-packages /ovms/python_deps
+COPY --from=build /usr/local/lib/python3.*/dist-packages/MarkupSafe-2.1.5.dist-info /ovms/python_deps/
+COPY --from=build /usr/local/lib/python3.*/dist-packages/jinja2* /ovms/python_deps/
+COPY --from=build /usr/local/lib/python3.*/dist-packages/markupsafe /ovms/python_deps/
```
Comment (mzegla) [inline Dockerfile.ubuntu:492]: not sure if dist info is required

**Review by mzegla — APPROVED** [review]

File: Dockerfile.redhat (hunk 1 above)
Comment (dtrawins) [reply Dockerfile.redhat]: it is license related

**Review by atobiszei — APPROVED** [review]

**Review by atobiszei — COMMENTED** [review]
Wouldn't droping all other site-packages impact UX/DX for using python nodes?

**Pull request merged**